app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -178,23 +179,29 @@ class MainActivity : AppCompatActivity(),
             }
         }
 
-        aztec.setOverlay(predicate, 0, ColorDrawable(0x80000000.toInt()), Gravity.FILL, attrs)
-        val progressDrawable = resources.getDrawable(android.R.drawable.progress_horizontal)
+        aztec.setOverlay(predicate, 0, ColorDrawable(0x80000000.toInt()), Gravity.FILL)
+        aztec.updateElementAttributes(predicate, attrs)
```

Comment:
Any particular reason to do the attribute setting in a separate call? With the separate call, the loop/search for the element is performed again, effectively doubling the CPU effort to find and alter the element. The `updateElementAttributes()` is a good API to have and expose to Aztec's client but, not sure if dropping the attribute updating from `setOverlay()` (and similar methods) is really necessary.